examples/sharedfilesystem/share_access/main.tf

```diff
@@ -0,0 +1,6 @@
+resource "vkcs_sharedfilesystem_share_access" "full_rw" {
```

Comment:
not really full since it's bounded to an ip

Reply:
will be rw_user